cowprotocol/cowswap
# feat: use bff proxies

# Summary

This PR makes use of BFF

## Coingecko
Get USD price estimation should use the proxy now, which added some caching headers

## Token List
Search for tokens not in the lists:
<img width="756" alt="image" src="https://github.com/cowprotocol/cowswap/assets/2352112/807a565d-f7bf-48a1-94e6-6d491776a3f1">

## Envs
Regarding the environment vars
* `BFF_BASE_URL` has been set to https://bff.barn.cow.fi in vercel. Also in dev and barn
* it has been set to https://bff.cow.fi in prod and staging


# Test
Coingecko:
- Input an amount, check the USD estimation request in the network tab, make sure the request is doe to https://bff.barn.cow.fi server

Tokens:
- Open token selection
- Search for SHIT
- Check a Pooo shows up 💩
